tests/parser/features/external_contracts/test_external_contract_calls.py

```diff
@@ -2452,3 +2452,63 @@ def do_stuff(f: Foo) -> uint256:
     c2 = get_contract(callee_code)
 
     assert c1.do_stuff(c2.address) == 1
+
+
+TEST_ADDR = b"".join(chr(i).encode("utf-8") for i in range(20)).hex()
+
+
+@pytest.mark.parametrize(
+    "typ,val",
+    [("address", TEST_ADDR), ("uint256", 2 ** 256 - 1), ("int128", 2 ** 127 - 1), ("bool", True)],
```

Comment:
i'm not sure it's necessary to test non-address types here. all we care about is the size of calldata, so we can just test a single type. (the uint256 case would fail for other reasons, namely that the data is OOB for address type)
```suggestion
    [("address", TEST_ADDR)],
```
```suggestion
    [("address", TEST_ADDR), ("uint256", 2 ** 256 - 1), ("int128", 2 ** 127 - 1), ("bool", True)],
```